docs/current/cli/465058-install.md

```diff
@@ -0,0 +1,129 @@
+---
+slug: /cli/465058/install
```

Comment:
@marcosnils can you please review the installation steps and confirm they are fine? I have assumed the same as the previous CLI.

Reply:
Yes, they are the same. I'll double check just to be sure